src/Controller/OgAdminRoutesController.php

```diff
@@ -46,7 +48,15 @@ public static function create(ContainerInterface $container) {
    * @return mixed
    *   List of available admin routes for the current group.
    */
-  public function overview($entity_type_id) {
+  public function overview(RouteMatchInterface $route_match) {
+
+    $parameter_name = $route_match->getRouteObject()->getOption('_og_entity_type_id');
```

Comment:
I was expecting that the [{node}](https://github.com/Gizra/og/pull/149/commits/1a7be5529f8fa8a0b4b224fe15880d0ce06656f0#diff-94902c9924afa2fe352c6170db324642R88) will be injected into controller, but for some reason we need to do this hack.

@weitzman do you happen to know the reason why we need this hack? (I copied this pattern from Devel module.)